src/directives/sortable.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * @fileoverview Provides a directive that can be used to change the order of
+ * items in an array.
+ * It can be used to change the order of layers in the map for example.
+ * It requires JQuery-UI Sortable.
+ *
+ * Example:
+ *
+ * <ul ngeo-sortable="ctrl.map.getLayers().getArray()">
+ *   <li ng-repeat="ctrl.map.getLayers().getArray()">
+ *      <span class="handle"></span>{{layer.get('name')}}
+ *   </li>
+ * </ul>
+ */
+
+goog.provide('ngeo.sortableDirective');
+
+goog.require('ngeo');
+
+
+/**
+ * @param {angular.$log} $log Angular log service.
+ * @param {angular.$timeout} $timeout Angular timeout service.
+ * @return {angular.Directive} The directive specs.
+ * @ngInject
+ */
+ngeo.sortableDirective = function($log, $timeout) {
+  return {
+    restrict: 'A',
+    scope: true,
+    link:
+        /**
+         * @param {angular.Scope} scope Scope.
+         * @param {angular.JQLite} element Element.
+         * @param {angular.Attributes} attrs Attributes.
+         */
+        function(scope, element, attrs) {
+          var attr = 'ngeoSortable';
+          var prop = attrs[attr];
+
+          var sortable = /** @type {Array} */ (scope.$eval(prop));
+
+          if (!angular.element.fn || !angular.element.fn.jquery) {
+            $log.error('ui.sortable: jQuery should be included before' +
+                'AngularJS!');
```

Comment:
We don't use the `$log` service elsewhere. Please use `goog.asserts.assert` instead. The advantage is that the compiler will remove the assertion.

```js
goog.asserts.assert(goog.isDef(angular.element.fn) && goog.isDef(angular.element.fn.jquery),
    'jQuery is required');
```